to-do-app/src/components/ListTitles/index.js

```diff
@@ -0,0 +1,51 @@
+import NewList from '../NewList';
+import '../../ToDoApp.css';
+import './style.css';
+import React, { Component } from 'react';
+import CircleOutlinedIcon from '@mui/icons-material/CircleOutlined';
+
+class ListTitles extends Component {
+  filterLists(title) {
```

Comment:
This function is filtering for active list only, right? I would rename to something like getActiveList.

Reply:
Yes, I was thinking about this too. I'll rename it.